tests/coalaTest.py

```diff
@@ -162,6 +163,16 @@ def test_show_all_bears(self, debug=False):
     def test_show_all_bears_debug(self):
         return self.test_show_all_bears(debug=True)
 
+    def test_show_all_bears_debug_exception(self, debug=True):
```

Comment:
Please use naming like `test_show_all_bears_error` or `test_show_all_bears_error_debug`. It's just more consistent

BTW: Is there already some test for error on collecting bears (without debug stuff)